config-templates/ibm/ibm-object-storage-plugin/2.2/custom-parameters.json

```diff
@@ -40,15 +40,15 @@
         "supported-values": ["IBM", "AWS", "Wasabi"]
     },
     {
-        "description": "Enter Object storage region. For more info, refer to https://ibm.biz/cos-storage-classes",
-        "displayname": "Object Storage Region",
+        "description": "Enter the region where your object storage is located. For IBM COS regions, see https://ibm.biz/cos-endpoints-list. For Wasabi, see https://ibm.biz/wasabi-endpoints. For AWS, see https://ibm.biz/aws-endpoints.",
+        "displayname": "Object Storage region",
         "name": "cos-storageclass",
         "category": "config",
         "required": "true"
     },
     {
-        "description": "Object storage service endpoint. Required when 'Object Storage Service Provider' is not set. Preference is given to 'Object Storage Service Provider' when both are set. For more information, refer to https://ibm.biz/cos-endpoints",
-        "displayname": "Object Storage Service Endpoint",
+        "description": "Object Storage service endpoint. Required when using object storage providers other than IBM, AWS, or Wasabi. Preference is given to 'Object Storage Service Provider' when both are set.",
```

Comment:
`Required when using object storage providers other than IBM, AWS, or Wasabi. Preference is given to 'Object Storage Service Provider' when both are set."` 
----> 
`Required when using Object Storage providers other than IBM, AWS or Wasabi. Preference is given to 'Object Storage provider' when both are set."`